.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+
+updates:
+  - package-ecosystem: "docker"
+    target-branch: "iroha2-dev"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+
+  - package-ecosystem: "github-actions"
+    target-branch: "iroha2-dev"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
```suggestion
```

Reply:
I would do not touch GH Actions updates since that sometimes a new Actions versions could change the module logic and broke the workflow then. For example, we encountered this with `convictional/trigger-workflow-and-wait` or  `java` Actions. By the way, it's fine to catch Actions updates automatically, but we have to check manually the context of those updates and as a rule, do not merge everything in one common PR.

Reply:
this is a fair point. However, `dependabot` will raise alert and only open a PR, never merge on it's own. We would still like to be aware of security vulnerabilities even if we don't want to fix it right away